server/views/pages/exhibition.njk

```diff
@@ -35,89 +35,97 @@
     <div class="exhibition-hero">
       {% componentV2 'picture', {images: exhibition.featuredImages}, {}, {'lazyload': true, extraClasses: ['exhibition-hero__picture']} %}
 
-      <div class="exhibition-hero__copy">
+      <div class="exhibition-hero__copy {{ {l:10} | spacingClasses({margin: ['bottom']}) }}">
         <div class="is-hidden-m is-hidden-l is-hidden-xl">
           {% component 'wobbly-edge', {background: 'white'} %}
         </div>
-        <div class="container bg-white {{ {m:0} | spacingClasses({margin: ['top', 'right', 'bottom', 'left'], padding: ['top', 'right', 'bottom', 'left']}) }}">
+
+        <div class="container {{ {m:0} | spacingClasses({padding: ['left']}) }}">
           <div class="grid">
             <div class="{{ {s:12} | gridClasses }}">
-
-              <div class="bg-white inline-block {{ {m:4} | spacingClasses({'padding': ['right', 'left']}) }} {{ {s:4} | spacingClasses({'padding': ['top', 'bottom']}) }}">
-                <div class="flex flex--v-center font-elf-green {{ {s:'HNM4', m:'HNM5', l:'HNM4'} | fontClasses }}">
-                  {% icon 'other/ticket', null, ['icon--2x', 'icon--teal'] %}
-                  <strong class="caps {{ {s: 2} | spacingClasses({margin: ['left']}) }} {{ {s: 6} | spacingClasses({margin: ['right']}) }}">Free</strong>
-                  {% if exhibition.end %}
+              <div class="bg-white inline-block exhibition-hero__box {{ {m:3, l:4} | spacingClasses({'padding': ['right', 'left']}) }} {{ {s:4} | spacingClasses({'padding': ['top', 'bottom']}) }} rounded-top-left rounded-bottom-right">
+                <div class="flex flex--v-center flex--h-space-between {{ {s:'HNM4', m:'HNM5', l:'HNM4'} | fontClasses }}">
+                  <span class="flex flex--v-center">
+                    {% icon 'other/ticket', null, ['icon--2x'] %}
```

Comment:
Passing around these icon classes makes me feel funny, but I'm not sure what we might do about it.